osbs/build/config_map_response.py

```diff
@@ -58,4 +59,6 @@ def get_data_by_key(self, name):
         if data is None or name not in data:
             return {}
 
+        if name.rsplit('.', 1)[-1] == 'yaml':
```

Comment:
Can we make this `in ('yaml', 'yml')` since both extensions are usually associate with YAML files?

Reply:
sure